BookReaderDemo/css/index.css

```diff
@@ -85,11 +85,22 @@
     color: black;
     z-index: 9999;
     font-size: 2.85em;
-    height: 95%;
-    margin: 50px;
-    padding: 25px;
-    background-color: #f8edc0;
+    height: 100%;
     border-radius: 8px;
     box-shadow: 0 0 50px 50px #f8edc0ff;
     overflow-y: scroll;
+  }
+
+  .translation .trParagraph {
```

Comment:
To keep the styling here consistent, giving the layer a `.BRtranslateLayer` class, and then the paragraphs/lines/etc can have the same classes (`.BRparagraphElement`, `.BRlineElement`, etc)

Reply:
The former :+1: Note we shouldn't need a `tr` prefix on the paragraphs/lines/etc, but can re-use the class from the textLayer :+1: